src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,21 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => {
+  if (todo.user) {
+    return (
+      <article className={`TodoInfo${todo.completed ? (' TodoInfo--completed') : ''}`}>
+        <h2 className="TodoInfo__title">
+          {todo.title}
+        </h2>
+        <UserInfo user={todo.user} />
+      </article>
+    );
+  }
+
+  return (<></>);
```

Comment:
Just don't render the UserInfo if you don't have the user. And restructure the todo, please
```suggestion
  return (
    <article className={`TodoInfo${todo.completed ? (' TodoInfo--completed') : ''}`}>
      <h2 className="TodoInfo__title">
        {title}
      </h2>
      {user && <UserInfo user={todo.user} />}
    </article>
  );
```